website/src/examples/carto-sql-layer.js

```diff
@@ -0,0 +1,112 @@
+import React, {Component} from 'react';
+import {GITHUB_TREE} from '../constants/defaults';
+import App from 'website-examples/carto-sql/app';
+
+import makeExample from '../components/example';
+
+const INDEXES = {
+  'Total Spend': 'txn_amt',
+  'Total Transactions': 'txn_cnt',
+  'Total Accounts': 'acct_cnt',
+  'Average Ticket Size': 'avg_ticket'
+};
+const INDUSTRIES = {
+  'Total Retail': 'ret',
+  'Total Apparel': 'app',
+  'Accommodations': 'acc',
+  'Automotive Fuel': 'aut',
+  'Eating Places': 'eap',
+  'Grocery and Food Stores': 'gro'
+};
+
+class CartoSQLDemo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      viewState: {
+        latitude: 40.7368521,
+        longitude: -73.9936065,
+        zoom: 11,
+        pitch: 60,
+        bearing: 0
+      }
+    };
+  }
+
+  componentDidMount() {
+    this.animate();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (!prevProps.params.rotate.value && this.props.params.rotate.value) {
+      this.animate();
+    }
+  }
+
+  animate() {
+    const enableAnimation = this.props.params.rotate.value;
+    if (enableAnimation) {
+      const {viewState} = this.state;
+      viewState.bearing += 0.03;
+      this.setState({ initialViewState: {...viewState }});
+      window.requestAnimationFrame(this.animate.bind(this));
+    }
+  }
+
+  static title = 'Mastercard Index In NewYork';
+  
+  static code = `${GITHUB_TREE}/examples/website/carto-sql`;
+
+  static parameters = {
+    index: {displayName: 'Index', type: 'select', options: Object.keys(INDEXES), value: 'Total Spend'},
+    industry: {displayName: 'Industry', type: 'select', options: Object.keys(INDUSTRIES), value: 'Total Retail'},
+    rotate: {displayName: 'Rotation animation', type: 'checkbox', value: true},
+  };
+
+  static mapStyle = 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json';
+
+  static renderInfo() {
+    return (
+      <div>
+        <p>Mastercard Index January 2020 using a z18 QuadGrid. A locations’s  index shows how it ranks against all the other locations in the area. The number represents the percentile, for example, a 900 spend index means a location has higher spend than 90% of locations.</p>
```

Comment:
The numeric threshold should be exposed in the legend. There also should be a tooltip showing the value on hover.